### PR TITLE
Create testwhitespace. (fix #11)

### LINKED ISSUE #11: abc

### PR DESCRIPTION
**Pull request by dla-kirito**
fix #11

### CODE REVIEW

File: testwhitespace.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+y
```
Comment (dla-kirito) [inline testwhitespace.yaml:1]: String<T>

File: testwhiyydsspace.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+y
```
Comment (zzuu666) [inline testwhiyydsspace.yaml:1]:
```suggestion
y

sb
```

File: testwhiyydsspace.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+
```
Comment (yuexi2021) [inline testwhiyydsspace.yaml:2]: test

Comment (yuexi2021) [reply testwhiyydsspace.yaml:2]: test

File: testwhiyydsspace.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+..
```
Comment (yuexi2021) [inline testwhiyydsspace.yaml:3]: test

**Review by yuexi2021 — CHANGES_REQUESTED** [review]
checkout my comments

**Review by yuexi2021 — APPROVED** [review]

File: testwhitespace.yaml (hunk 1 above)
Comment (yuexi2021) [reply testwhitespace.yaml]: test

Comment (yuexi2021) [reply testwhitespace.yaml]: test

Comment (yuexi2021) [reply testwhitespace.yaml]: attest

Comment (yuexi2021) [reply testwhitespace.yaml]: test

**Review by yuexi2021 — APPROVED** [review]

**Review by yuexi2021 — APPROVED** [review]

File: testwhiyydsspace.yaml (hunk 2 above)
Comment (dla-kirito) [reply testwhiyydsspace.yaml:4]:
```suggestion
y h
```

File: testwhiyydsspace.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (dla-kirito) [inline testwhiyydsspace.yaml:1]:
```suggestion
1111
```

**Pull request closed without merging**